x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations.tsx

```diff
@@ -0,0 +1,178 @@
+
+  if (dataLoading && !curations.length) return <Loading />;
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>{CURATIONS_OVERVIEW_TITLE}</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+        <EuiPageHeaderSection>
+          <EuiButtonTo to={generateEnginePath(ENGINE_CURATIONS_NEW_PATH)} fill>
+            {CREATE_NEW_CURATION_TITLE}
+          </EuiButtonTo>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <FlashMessages />
+        <CurationsTable />
+      </EuiPageContent>
+    </>
+  );
+};
+
+export const CurationsTable: React.FC = () => {
+  const { dataLoading, curations, meta } = useValues(CurationsLogic);
+  const { onPaginate, deleteCurationSet } = useActions(CurationsLogic);
+
+  const columns: Array<EuiBasicTableColumn<Curation>> = [
+    {
+      field: 'queries',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.curations.table.column.queries',
+        { defaultMessage: 'Queries' }
+      ),
+      render: (queries: Curation['queries'], curation: Curation) => (
+        <EuiLinkTo
+          data-test-subj="CurationsTableQueriesLink"
+          to={generateEnginePath(ENGINE_CURATION_PATH, { curationId: curation.id })}
+        >
+          {queries.join(', ')}
+        </EuiLinkTo>
+      ),
+      width: '40%',
+      truncateText: true,
+      mobileOptions: {
+        header: true,
+        // Note: the below props are valid props per https://elastic.github.io/eui/#/tabular-content/tables (Responsive tables), but EUI's types have a bug reporting it as an error
+        // @ts-ignore
+        enlarge: true,
+        width: '100%',
+        truncateText: false,
+      },
+    },
+    {
+      field: 'last_updated',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.curations.table.column.lastUpdated',
+        { defaultMessage: 'Last updated' }
+      ),
+      width: '30%',
+      dataType: 'string',
+    },
+    {
+      width: '120px',
+      actions: [
+        {
+          name: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.editAction',
+            { defaultMessage: 'Edit' }
+          ),
+          description: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.editAction',
+            { defaultMessage: 'Edit' }
+          ),
+          type: 'icon',
+          icon: 'pencil',
+          color: 'primary',
+          onClick: (curation: Curation) => {
+            const { navigateToUrl } = KibanaLogic.values;
+            const url = generateEnginePath(ENGINE_CURATION_PATH, { curationId: curation.id });
+            navigateToUrl(url);
+          },
+          'data-test-subj': 'CurationsTableEditButton',
+        },
+        {
+          name: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.deleteAction',
```

Comment:
This translation looks like it's duplicated. on the next link for description. Should we pull this out into a constant?

Reply:
Alternatively, if these have different purposes, use different keys!

Reply:
👍 I think I'll opt to use different keys and have the description be more descriptive

Reply:
https://github.com/elastic/kibana/pull/91565/commits/78249d01334b6b059b482f338294b086ba918326